.gitignore

```diff
@@ -189,6 +189,10 @@ $RECYCLE.BIN/
 *.lnk
 
 ### PyCharm ###
+
+# Jetbrains IDE configs
+.idea/
```

Comment:
Can presumably remove all of the individual `.idea/**/...` entries below since they're covered by this blanket entry now?